src/SmartCommander/Views/FilesPane.axaml.cs

```diff
@@ -29,7 +36,31 @@ public FilesPane()
         private void OnLoaded(object sender, RoutedEventArgs e)
         {
             var PaneDataGrid = this.Get<DataGrid>("PaneDataGrid");
-            PaneDataGrid.Focus();
+            if (PaneDataGrid != null)
+            {
+                focusManager = TopLevel.GetTopLevel((Visual)PaneDataGrid)?.FocusManager;
+                PaneDataGrid.AddHandler(KeyDownEvent, dataGrid_PreviewKeyDown, RoutingStrategies.Tunnel);
+                PaneDataGrid.Focus();
+            }
+        }
+
+
+
+        public void dataGrid_PreviewKeyDown(object? sender, KeyEventArgs e)
```

Comment:
fix backspace and enter behaviour when we renamin file or folder inside Datagrid